backend/templates/emails/reminder-email.html

```diff
@@ -0,0 +1,84 @@
+<h2>Reminder to Renew <b>{{ name }}</b> on Penn Clubs & SAC Fair Signup by 8/24!</h2>
+<p style="font-size: 1.2em">
+    Dear <b>{{ name }}</b> leaders,
+</p>
+
+<p style="font-size: 1.2em">
+    You are receiving this notification because you were the contact for your organization during the 2019 SAC
+    Activities Fair. If you have already graduated, <b>PLEASE FORWARD THIS EMAIL</b> to the current leadership of
+    the group so that they may complete the registration process and sign up for the SAC Fair. If
+    they are not currently listed as an officer for your club on Penn Clubs, they will need to
+    email <b>contact@pennclubs.com</b> with their <b>name, PennKey, and club name</b>.
+</p>
+
+<p style="font-size: 1.2em">
+    We have opened the student group registration process for the 2020-2021 academic year. To be
+    considered an active student group on campus, the University requires all student organizations
+    to be registered with and approved by the Office of Student Affairs. <b>This year, OSA has migrated the Student
+        Groups platform from the previous system, G.O. Penn, to Penn Labs' platform, Penn Clubs.</b>
+    All official registry of student groups, including annual student group registration process, will now be handled
+    through Penn Clubs.
+</p>
+
+<p style="font-size: 1.2em">
+    Additionally, the Student Activities Council (SAC) is partnering with Penn Clubs to host a
+    virtual multiday Activities Fair from <b>Sep 1 – 3</b>. More details on the Fair will be shared at a
+    later date by SAC. To participate in the Fair, you must be a student-run club at the
+    University of Pennsylvania approved by the OSA. Registration through the Penn Clubs portal will require PennKey
+    authentication.
+    The deadline to sign up for the SAC Fair is <b>August 24th</b>.
+</p>
+
+<p style="font-size: 1.2em">
+    Once the request for registration is approved, the group will be listed on the
+    front page of Penn Clubs and will be searchable for students on the site.
+</p>
+
+<p style="font-size: 1.2em">
+    <b>IMPORTANT NOTE:</b> Since this information is being distributed through various listservs and methods, please
+    communicate
+    with your entire executive board before signing up for the fair. <b>ONLY ONE</b> officer of the club
+    should be designated to register the group.
+
+</p>
+
+<p style="font-size: 1.2em">
+    <u><b>Re-Registration Process (including SAC Fair sign-up) - Deadline Aug. 24th</b></u>
+</p>
+
+<p style="font-size: 1.2em">
+    Again, it is imperative that your group designate <b>ONLY ONE</b> executive board member to complete this process,
+    preferably a
+    member who is listed on pennclubs.com as an officer for your club.
+</p>
+
+<p style="font-size: 1.2em">
+    To re-register a returning student group, you can access the Penn Clubs renewal site at <a
+        href="https://pennclubs.com/club/pennlabs/renew">https://pennclubs.com/club/pennlabs/renew</a>.
```

Comment:
Since this is a personalized club email based on the club, we can actually link them directly to the club renewal page instead of the generic page to save them a few clicks.

You can use the large green button to direct the user's attention to the action item. See the [the original email](https://pennclubs.com/api/emailpreview/?email=renew) for details.